app/src/main/java/com/safetyheads/akademiaandroida/utils/FullNameVaildator.kt

```diff
@@ -0,0 +1,67 @@
+package com.safetyheads.akademiaandroida.utils
+
+import android.content.Context
+import android.text.InputFilter
+import android.widget.EditText
+import androidx.core.widget.addTextChangedListener
+import com.safetyheads.akademiaandroida.R
+
+/*
+* To use this validator you have to add to your code:
+* FullNameValidator.attach(fullNameEditText, this)
+*/
+
+object FullNameValidator {
```

Comment:
Something is not right, when I type "ABC" then it returns that surname is ok even if there is no surname